### PR TITLE
Ravatars

### PR DESCRIPTION
**Pull request by rsegeda**
Tweet of Policy ID: 
https://twitter.com/roman_segeda/status/1490624186208956420

### CODE REVIEW

**Comment by KaiNFT** [comment]
Dear Contributor, 

The policy ID included in your file does not match with the policy ID listed on your social media/website.

File: projects/rsegeda [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "project": "^[A-Z](avatar)$",
+    "policies": [
+      "66c97d01c147c16c1e7344ee349e9c1a2dab97ebd9c7aaac9f5b4e61"
```
Comment (rsegeda) [inline projects/rsegeda:5]:
```suggestion
      "38620cdffe114577057b13387325d9682b136e95f9709cfdaf33bd07"
```

**Comment by rsegeda** [comment]
> Dear Contributor,
> 
> The policy ID included in your file does not match with the policy ID listed on your social media/website.

@KaiNFT 🤦 , fixed

**Comment by KaiNFT** [comment]
Dear Contributor, 

Your social media is quite new, do you have another form of social media or a website I can use to verify you?

**Comment by rsegeda** [comment]
> Dear Contributor,
> 
> Your social media is quite new, do you have another form of social media or a website I can use to verify you?

https://github.com/rsegeda/avatars/blob/main/README.md

Is this enough?

**Comment by KaiNFT** [comment]
Dear Contributor, 

You must have a valid website that's directly linked to your project in order for me to verify this request.

**Comment by rsegeda** [comment]
> Dear Contributor,
> 
> You must have a valid website that's directly linked to your project in order for me to verify this request.

If I don't use other social media, where should I post it? What websites are considered `valid`? I created a Twitter account recently. However, it's not old enough. I have my GitHub account, which is relatively mature, and it is not `valid`, but why? What is wrong with my repository?

**Comment by KaiNFT** [comment]
You need a website that is based around the project itself, it needs to show the NFT and each policy ID.

**Comment by rsegeda** [comment]
I give up...

**Pull request closed without merging**